bluemira/equilibria/equilibrium.py

```diff
@@ -324,8 +330,10 @@ def from_eqdsk(
             from_cocos=from_cocos,
             full_coil=full_coil,
             qpsi_positive=qpsi_positive,
+            regrid_nx_nz=regrid_nx_nz,
             **kwargs,
         )
+
```

Comment:
Please could we include something along the lines of:
```suggestion
if regrid_nx_nz is not None:
    bluemira_info("NOTE: Loading an Equilibrium from a file using a different grid will result in changes with respect to the original data.")
```

Could probably also do with a warning if nx, nz == nx, nz [existing] and no re-gridding action taken.